extensions/firebase-import-export/src/firebase-export.tsx

```diff
@@ -0,0 +1,48 @@
+import { getPreferenceValues, showHUD, closeMainWindow } from "@raycast/api";
+import { backups, initializeFirebaseApp } from "firestore-export-import";
+import { IExportOptions } from "firestore-export-import/dist/helper";
+import moment from "moment";
+import { convertStringToNumber, readFile, writeFile } from "./utils";
+import { Preferences } from "./interfaces";
+
+/* Getting the preferences from the Raycast API. */
+const preferences = getPreferenceValues<Preferences>();
+
+/* Setting the export options. */
+const exportOptions: IExportOptions = {
+  docsFromEachCollection:
+    preferences.limitDocumentExport?.length !== 0 ? convertStringToNumber(preferences.limitDocumentExport) : 0,
+};
+
+/* Initializing the Firebase app with the service account. */
+initializeFirebaseApp(JSON.parse(readFile(preferences.firebaseAuth)));
+
+const collectionsList: Array<string> | undefined =
+  preferences.collectionList?.length !== 0 ? preferences.collectionList.split(",") : undefined;
+
+export default async function Command() {
+  try {
+    /* It closes the main window of the Raycast application. */
+    await closeMainWindow({ clearRootSearch: true });
+    /* Exporting the data from the Firestore database. */
+    await backups(collectionsList, exportOptions).then((collections: any) => {
+      if (preferences.fileByCollection) {
+        Object.entries(collections).forEach((entry) => {
+          const [collection, content] = entry;
+          writeFile(
+            JSON.stringify({ [collection]: content }, null, 4),
+            preferences.exportFileStorage,
+            collection + "_" + moment().format("YYYY-MM-DD_Hmm") + ".json"
```

Comment:
Let's avoid using `moment` as the [package cost is pretty high](https://bundlephobia.com/package/moment@2.29.4), you can use some alternatives like date-fns or dayjs.